### PR TITLE
Add endpoint to the python integration tests

### PR DESCRIPTION
**Pull request by evadb**
This PR adds extract_endpoint to the python_integration tests on circleCI, yay!

Unfortunately this changes the reader used from LocalReader to AzureReader, which currently breaks the node integration tests :( But! The python integration portion successfully runs. 

To look into: currently requires a sleep statement because processes are required to run in the background, which CircleCI doesn't care about and just keeps going, breaking all the things. Maybe there's a better, more reliable way of doing this?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -199,37 +199,94 @@ jobs:
...
-          name: "Loading data"
+          name: "Extracting data"
+      - run:
+          command: |
+              cd extract_endpoint
+              . ~/.venv/bin/activate
+              extract_endpoint upload ../etl/tests/randomized_energuide_data.zip 2018-03-14
+          name: "Send data to azure, trigger TL process"
+      - run:
+          sleep 5
```
Comment (buckley-w-david) [inline .circleci/config.yml]: Should we wait until #386 is merged in so we can remove this sleep and spin until we get a status that it's done?

File: extract_endpoint/tests/check_mongo.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ sleep 5
+ let TIMEOUT=TIMEOUT+1
+ if [ $TIMEOUT -eq 6 ]; then
+    echo 'process timed out, mongo was not populated'
+    echo 'exiting tests'
+    exit 1
+ fi
+ RESULT=$(mongo energuide --eval 'db.dwellings.count()' --quiet)
+done
+echo 'mongo populated, continuing process'
```
Comment (sastels) [inline extract_endpoint/tests/check_mongo.sh]: add a new line at the end of the file

File: .circleci/config.yml [hunk 3]
```diff
@@ -286,7 +286,8 @@ jobs:
               extract_endpoint upload ../etl/tests/randomized_energuide_data.zip 2018-03-14
           name: "Send data to azure, trigger TL process"
       - run:
-          sleep 5
+          command: "./check_mongo.sh"
```
Comment (sastels) [inline .circleci/config.yml]: 👍

File: .circleci/config.yml [hunk 4]
```diff
@@ -199,37 +199,97 @@ jobs:
...
+              sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+              sudo apt-get install apt-transport-https
+              sudo apt-get update && sudo apt-get install azure-cli
           name: "Installing packages"
       - run:
           command: |
+              echo $EXTRACT_ENDPOINT_STORAGE_DOMAIN
+              echo $EXTRACT_ENDPOINT_STORAGE_ACCOUNT
+              echo $EXTRACT_ENDPOINT_STORAGE_KEY
+              echo $EXTRACT_ENDPOINT_CONTAINER
```
Comment (sastels) [inline .circleci/config.yml]: why the echos?

File: api/test/enum.test.js [hunk 5]
```diff
@@ -28,9 +28,9 @@ describe('Enum values', () => {
   const testFields = {
     dwellingHouseId: { testValue: 189250 },
     dwellingYearBuilt: { testValue: 1900 },
-    dwellingCity: { testValue: 'Charlottetown' },
-    dwellingRegion: { testValue: 'PE' },
-    dwellingForwardSortationArea: { testValue: 'C1A' },
+    dwellingCity: { testValue: 'Dartmouth' },
```
Comment (sastels) [inline api/test/enum.test.js]: why did you have to change all these javascript files?

Comment (evadb) [reply api/test/enum.test.js]: I missed the warm embrace of javascript

File: .circleci/config.yml (hunk 4 above)
Comment (evadb) [reply .circleci/config.yml]: because I totally forgot about them 😂 thanks! 🔥

**Comment by buckley-w-david** [comment]
😱

**Pull request merged**